src/lib/storage/StorageApi.ts

```diff
@@ -233,17 +244,18 @@ export class StorageApi {
   }
 
   /**
-   * Delete multiple files on the same bucket
-   * @param bucketName
-   * @param paths the relative file paths to be deleted excluded the bucket name
+   * Deletes multiple files within the same bucket
+   *
+   * @param bucketId The bucket which contains the files.
+   * @param paths An array of files to be deletes, including the path and file name. For example [`folder/image.png`].
```

Comment:
it looks like the user doesn't have to include the file name right?

For example, if they put `/folder`, it would delete the entire folder?

Reply:
No. The user will have to get the full path to delete. Having a separate endpoint to delete a folder is on the roadmap.